{"uuid": "46424e79-3a14-4e3e-8fcc-6d5a7eb0c5be", "children": ["5d5b3935-2303-4d77-a0ea-7f2bdff98c13"], "befores": [{"name": "state_driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: Failed to create a Chrome process.\n  (Driver info: chromedriver=2.44.609545 (c2f88692e98ce7233d2df7c724465ecacfe74df5),platform=Mac OS X 10.14.0 x86_64)\n\n", "trace": "  File \"/Users/kouhui/Documents/python/ui-automation/venv/lib/python3.6/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/kouhui/Documents/python/ui-automation/venv/lib/python3.6/site-packages/_pytest/fixtures.py\", line 957, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/kouhui/Documents/python/ui-automation/venv/lib/python3.6/site-packages/_pytest/fixtures.py\", line 787, in call_fixture_func\n    res = fixturefunc(**kwargs)\n  File \"/Users/kouhui/Desktop/自动化文件/ui-automatic/conftest.py\", line 15, in state_driver\n  File \"/Users/kouhui/Documents/python/ui-automation/venv/lib/python3.6/site-packages/selenium/webdriver/chrome/webdriver.py\", line 81, in __init__\n    desired_capabilities=desired_capabilities)\n  File \"/Users/kouhui/Documents/python/ui-automation/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/Users/kouhui/Documents/python/ui-automation/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py\", line 252, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/Users/kouhui/Documents/python/ui-automation/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/kouhui/Documents/python/ui-automation/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1577697347891, "stop": 1577697348933}], "start": 1577697347891, "stop": 1577697349099}